cmd/promtool/main.go

```diff
@@ -191,6 +191,16 @@ func checkFileExists(fn string) error {
 	return err
 }
 
+func addScheme(urlString string) (string, error) {
```

Comment:
Can we have tests for this function?

Reply:
Sure. While adding one, I was looking at [main_test.go](https://raw.githubusercontent.com/prometheus/prometheus/master/cmd/promtool/main_test.go), found that it's setting up a mock server and testing query range with this server. The only problem is that for our use case, the mock server returns a URL that always has a schema(and hence is not a valid test). Should I manually edit out the schema from the URL argument of type server in [httptest](https://golang.org/pkg/net/http/httptest/)? If not, how should I write this test?

Reply:
just a simple test of addScheme helper function in its own; not starting a prometheus function

Reply:
I have added a test in [bd40935](https://github.com/prometheus/prometheus/commit/bd40935a69d7ecad3e7139bb884318a5bb0885ba)